docs/developer/payments.mdx

```diff
@@ -0,0 +1,524 @@
+---
+title: Payments
```

Comment:
Why isn't this called `Transactions`?

Reply:
@witoszekdev  In core, we have old flow that uses `Payment` types and the new one, that uses `Transaction`. Both of them are related to processing payments. Someone who is not familiar with Saleor, will try to find `Payments` chapter.